### PR TITLE
Remove 'Featured in' logo block from VPN landing page (Fixes #11600)

### LINKED ISSUE #11600: Remove VPN "featured in" logo block
## Description

URL: https://www.mozilla.org/en-US/products/vpn/

License agreements for 3 out of the 4 logos run out this month. Ask from legal is to remove the logo block until we either renew contracts or update with new logos.

image

---

#### Success Criteria
- [ ] Featured logo block is removed.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Legal have asked that we remove these logos for now as the trademark agreements for most of them expire shortly (one this week).

## Issue / Bugzilla link

#11600

## Testing

http://localhost:8000/en-US/products/vpn/

### CODE REVIEW

File: bedrock/products/templates/products/vpn/includes/featured-press.html [hunk 1]
```diff
@@ -1,15 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-<aside class="vpn-featured-press-logos">
-  <h3 class="vpn-featured-press-heading">{{ ftl('vpn-shared-featured-in') }}</h3>
```
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/includes/featured-press.html:8]: **question (non-blocking, l10n):** if we were quite certain this block was not coming back, would we remove the ftl translation? or just mark is as outdated?

**Review by maureenlholland — CHANGES_REQUESTED** [review]
I did a double take at the number of files, because I didn't expect so many but all the img variations add up. Looks good for VPN Landing 👍 

There are still some references to the deleted images in Firefox welcome pages to clean:
https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/welcome/page9.de.html#L49
https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/welcome/page9.fr.html#L49

File: bedrock/products/templates/products/vpn/includes/featured-press.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/products/templates/products/vpn/includes/featured-press.html:8]: The block might come back, so I'd say we should keep the strings for now

**Comment by alexgibson** [comment]
Great catches @maureenlholland, updated thanks

**Review by maureenlholland — APPROVED** [review]
LGTM r+ 🧹

**Pull request merged**